src/config.h

```diff
@@ -112,6 +112,8 @@ typedef struct {
   // Can allow to control the seperation between phrases in different array slots (related to the SLOP parameter in ft.search command)
   // Default value is 100. 0 will not increment (as if all text is a continus phrase).
   unsigned int multiTextOffsetDelta;
+  // bitarray of dialects used by all indices
+  uint_least8_t used_dialects;
```

Comment:
We can keep it to support extraordinary architectures, but since `uint8_t` is already widely used, extraordinary architectures would be broken anyway if they have no `uint8_t`.

Reply:
Yes, in all archs that contain `uint8_t`, `uint_least8_t` is identical to it. So there's no loss there. This is just a habit of mine because the std doesn't mandate that `uint8_t` exist, but it does mandate `uint_least8_t`.